packages/components/src/components/tab/tab.ts

```diff
@@ -111,6 +111,12 @@ export default class SdTab extends SolidElement {
         border-bottom: none;
         border-radius: 4px 4px 0 0;
       }
+
+      .tab--bottom-border::before {
+        content: '';
+        @apply absolute w-full bottom-0 border-b border-neutral-400 invisible hover:visible;
+        height: 1px;
+      }
```

Comment:
Hmm. Usually we would create the element as an own DOM element via JS here. What do you think?